scripts/generateAssetData/generateRelatedAssetIndex.ts

```diff
@@ -0,0 +1,26 @@
+import 'dotenv/config'
+
+import assert from 'assert'
+
+import { generateRelatedAssetIndex } from './generateRelatedAssetIndex/generateRelatedAssetIndex'
+
+const main = async () => {
+  try {
+    // check zerion api key is set before starting, prevents getting through generateAssetData()
+    // and then failing
+    assert(
+      process.env.ZERION_API_KEY !== undefined,
+      'Missing Zerion API key - see readme for instructions',
+    )
+
```

Comment:
we don't really need this boilerplate here if we don't want, suppose it doesn't hurt, but comment isn't really relevant in this context

Reply:
This can be used as a standalone script and *will* fetch from Zerion, so might as well keep it! 8e1308df86 for comment cleanup that actually reflects what this does

Reply:
Right, just saying we do this same assertion in `generateRelatedAssetIndex/generateRelatedAssetIndex.ts` so not really a big deal for this script compared to `generateAssetData.ts`